scripts/west_commands/tests/test_stm32cubeprogrammer.py

```diff
@@ -0,0 +1,400 @@
+# Copyright (c) 2020 Teslabs Engineering S.L.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+from pathlib import Path
+from unittest.mock import patch, call
+
+import pytest
+
+from runners.stm32cubeprogrammer import STM32CubeProgrammerBinaryRunner
+from conftest import RC_KERNEL_HEX, RC_KERNEL_ELF
+
+
+CLI_PATH = Path("STM32_Programmer_CLI")
+"""Default CLI path used in tests."""
+
+HOME_PATH = Path("/home", "test")
+"""Home path (used for Linux system CLI path)."""
+
+PROGRAMFILES_PATH = Path("C:", "Program Files")
```

Comment:
> Could you list all the official paths? In case the user decides to install to a custom location the --cli option is available.

I sent email but have no response up to now.

What about your path "c:\Program Files (x86)" ?

Reply:
I had 2.1.0 installed. Uninstalled and downloaded latest 2.5.0, it still suggests x86 path:

![image](https://user-images.githubusercontent.com/25011557/96846663-7e134d00-1452-11eb-938a-4c4e344db3e9.png)

I'm on Windows 10 1903 x64

![image](https://user-images.githubusercontent.com/25011557/96846766-9d11df00-1452-11eb-86a6-d93bb6eec469.png)

Reply:
> What about your path "c:\Program Files (x86)" ?

I mean:
Why not tackling both known paths : 
"c:\Program Files\STMicroelectronics\STM32Cube\STM32CubeProgrammer"
and "c:\Program Files (x86)\"
(by checking existence of cube programmer in one or both paths ?)